sceptre/resolvers/project_variables.py

```diff
@@ -24,6 +26,12 @@ def resolve(self):
         :returns: Value at argument location.
         :rtype: str
         """
+        init(autoreset=True)
+        simplefilter('always', DeprecationWarning)
```

Comment:
Could we use double quotes (") instead of single (') - both are equally valid and which one you choose is a personal preference but we use doubles elsewhere in the codebase.